presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedTestBase.java

```diff
@@ -203,14 +203,14 @@ protected QueryRunner createQueryRunner()
     public void testDeleteOnV1Table()
     {
         // Test delete all rows
-        long totalCount = (long) getQueryRunner().execute("CREATE TABLE test_delete with (format_version = '1') as select * from lineitem")
+        long totalCount = (long) getQueryRunner().execute("CREATE TABLE test_delete with (\"format-version\" = '1') as select * from lineitem")
```

Comment:
Can we add some tests with the old names in which they should also succeed but with a warning?

Reply:
That's what these two tests are for: https://github.com/prestodb/presto/blob/d1ac6aaaaf3acdc136a4e82d969f1886bccba211/presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedSmokeTestBase.java#L1948-L1990